### DISCUSSION TITLE
DNS_PROBE_FINISHED_NXDOMAIN error on `*.vercel.app`

### DISCUSSION THREAD

**Discussion by henrynoowah** [vercel/community#6800] [Bug Report]
### Page to Investigate

https://swr.vercel.app/

### Steps to Reproduce

I tried to access my personal projects deployment on https://noowah.vercel.app, and encountered this issue.

I checked other sites using `*.vercel.app` and checked that all sites using `*.vercel.app` domains having similar issue

example:
- https://swr.vercel.app/

**Comment by bhyoo99** [comment]
I'm having the same problem too.
In the same project, accessing via a custom domain works fine.

**Comment by henrynoowah** [comment]
- https://github.com/orgs/vercel/discussions/6807#discussioncomment-9395299

> Hi, guys, I just checked and want to let y'all know that it is now working

Ps. Closing as further discussion will should be done on the more `active` post
- https://github.com/orgs/vercel/discussions/6807#discussioncomment-9395299
